index.js

```diff
@@ -0,0 +1,72 @@
+const { mungeLocation, mungeWeather, mungedTrails } = require('./utils.js');
+const dotenv = require('dotenv');
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const request = require('superagent');
+
+const PORT = process.env.PORT || 3001;
+
+
+
+dotenv.config();
```

Comment:
you want to initiate this before you use your `.env`, which you are using on line 8. So be sure that `dotenv.config()` is at the top of your file.